### ISSUE TITLE
JAWS doesn't read toolbar option labels in Chrome/Edge

### ISSUE THREAD

**Issue by marjon4** [ckeditor/ckeditor5#10766]
## 📝 Provide detailed reproduction steps (if any)

1. Launch a Chrome or Edge browser window in Windows
2. Navigate to a webpage with CKEditor implemented
2. Activate JAWS screen reader
2. Navigate to CKEditor text editor by tab
3. Navigate to toolbar options

### ✔️ Expected result

JAWS should read "bold, button", if focus is on the bold formatting option in the toolbar.

### ❌ Actual result

JAWS only reads "button", which leaves screen reader users unaware of what the button does.

### ❓ Possible solution

A possible solution that seems to work better cross browser in screen readers is to put aria-label="bold" directly on the button-element instead of an aria-labelledby reference like the current solution. Haven't tested it thoroughly though.

## 📃 Other details

* Browser: Chrome/Edge
* OS: Windows 10
* First affected CKEditor version: 28.0.0
* Installed CKEditor plugins: Bold, Italic, Paragraph, PasteFromOffice

**Comment by bbenjamin** [comment]
This does not seem to be a problem for me on CKEditor5 34 
**JAWS w/speech history showing it correctly read the labels**
<img width="585" alt="jaws-log" src="https://user-images.githubusercontent.com/1742345/169883181-fc8f69bd-a66e-488f-b24c-7fdafe9c56c4.png">
JAWS rotor also demonstrating it correctly sees labels
<img width="622" alt="jaws-rotor" src="https://user-images.githubusercontent.com/1742345/169883270-3ce15790-7ce8-47f5-aec6-5d70723b3f6b.png">
Here's my version of JAWS
<img width="302" alt="jaws-version" src="https://user-images.githubusercontent.com/1742345/169883323-c16bddb7-0014-4d85-8f83-e5d7f6205d44.png">

**Comment by Comandeer** [comment]
I can't reproduce the issue. Buttons are read correctly by JAWS, e.g. "Bold toggle button".
